### PR TITLE
chore(Python): added equalize the array

### PR DESCRIPTION
**Pull request by Tanmoy07tech**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

Algorithm name is Equalize the array . I implement this algorithm with python programming
**Other information:**

### CODE REVIEW

**Comment by atin** [comment]
Implementation looks good! Though you can change the variable names a bit to make it more readable for eg. `freq` (or `frequency`) instead of `hast_table` and similarly for `max_hash_table`

Also you can shorten the code by using some built-in python functions, though its completely your choice
```python
def equalizeArray(arr):
    mode = max(arr, key=arr.count)    # element with max number of occurrence in the array i.e. mode of array
    num_mode = arr.count(mode)
    deletions = len(arr) - num_mode
    return deletions
```

File: algorithms/Python/arrays/equalize_the_array.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# arr=[1,2,2,3]
+# Delete the  2 elements 1 and 3 leaving arr=[2,2]. If both twos plus either the 1 or the 3 are deleted, it takes 3 deletions to leave either [3] or [1]. The minimum number of deletions is 2.
+
+#!/bin/python3
+
+import math
+import os
+import random
+import re
+import sys
```
Comment (atin) [inline algorithms/Python/arrays/equalize_the_array.py:12]: @Tanmoy07tech You can remove these imports since these are not used in the code.

**Review by atin — CHANGES_REQUESTED** [review]

File: algorithms/Python/arrays/equalize_the_array.py (hunk 1 above)
Comment (atin) [reply algorithms/Python/arrays/equalize_the_array.py:12]: P.S. - Also add this file in [README.md](https://github.com/MakeContributions/DSA/blob/main/algorithms/Python/README.md)

**Comment by ming-tsai** [comment]
Hi @Tanmoy07tech,
Any updated?

**Comment by ming-tsai** [comment]
Hi @Tanmoy07tech,
This pull request is closed due to is inactive, any update please re-open the pr

**Pull request closed without merging**